listenbrainz/webserver/static/css/follow.less

```diff
@@ -0,0 +1,73 @@
+@purple: #353070;
+@white: #ffffff;
+
+#follow-button {
+  margin-left: 10px;
+  font-family: "Sintony";
+
+  &.icon-only {
+    > .text {
+      margin-left: 4px;
+      width: 0px;
+      display: inherit;
+      vertical-align: inherit;
+      transition: width 0.42s;
+      overflow-x: hidden;
+    }
+    &:hover > .text,
+    &:focus > .text {
+      width: 4.5em;
+    }
+  }
+
+  &.block {
+    color: @purple;
+    background: @white;
+    border: 2px solid @purple;
+    font-weight: bold;
+    border-radius: 6px;
+    padding: 5px;
+    text-align: center;
+    width: 85px;
+
+    &.btn-info {
+      color: @white;
+      background: @purple;
+    }
+
+    svg {
+      display: none;
+    }
+  }
+}
+
+.follower-following-pills {
+  padding-bottom: 10px;
+}
+
+.follower-following-list {
+  height: 250px;
+  padding: 10px;
+  overflow-y: scroll;
+  border-radius: 2px;
+  box-shadow: inset 0px 11px 8px -10px #ccc;
+
+  > div {
+    display: flex;
+    align-items: center;
+    padding: 5px;
+    margin: 5px 10px;
+
+    > #follow-button {
```

Comment:
FOllow button in itself is a separate component and has id. I am a little confused. Should we change it to class?

Reply:
Yes, if there are multiple items sharing style, they should use a class rather than an id.
From [W3C docs](https://www.w3schools.com/htmL/html_id.asp): 
> You cannot have more than one element with the same id in an HTML document

Sure, the page won't catch on fire or anything, but let's keep it clean.

Reply:
done